packages/ui/styles/globals.css

```diff
@@ -42,12 +42,18 @@
     --rust: #8d5728;
   }
 
+  #popup-root {
+    width: 400px;
+    height: 600px;
+    @apply bg-card-radial bg-opacity-20;
```

Comment:
This color isn't quite right. See the onboarding cart. It's lighter and more gray. You can compare to penumbra.zone's cards as well.